scrapers/nfl_fantasy.py

```diff
@@ -46,53 +49,38 @@ def scrape_nfl_fan(limit):
                     unicode('0') if x == '-' else x, [x.text for x in row])
             if any(map(lambda x: x in player_stats[0], ['DEF'])):
                 name = player_stats[0].split('DEF')[0].rstrip()
-                projected_points = calculate_ppr('DEF', player_stats)
+                projected_points = \
+                    calculate_ppr('DEF',
+                                  convert_nfl_fantasy_data('DEF',
+                                                           player_stats))
                 hold.append([name, projected_points])
     return hold
 
 
-def calculate_ppr(pos, row):
-    projected_points = 0
+def convert_nfl_fantasy_data(pos, row):
+    stat_dict = generate_empty_stat_dict(pos)
```

Comment:
I would still recommend creating a separate class where you can pass in a bunch of arguments from the row. You can do some type of inheritance based on what the position is. I'm thinking something like this:

```
kwargs = {...} # all your keyword arguments from the row, like {touch_downs: row[6]}
row_class_dict = {
    'DEF': DefenseProjections(**kwargs),
    'QB': QBProjections(**kwargs),
    'TE': TEProjections(**kwargs)
}
```

This pattern is from that Martin Fowler book [I showed you a while back](http://refactoring.com/catalog/replaceConditionalWithPolymorphism.html). Craig was pretty big on it.